automation_tools/satellite6/capsule.py

```diff
@@ -212,7 +212,7 @@ def generate_capsule_certs(capsule_hostname):
     """
     cert_path = '{}-certs.tar'.format(capsule_hostname)
     result = run('[ -f {} ]'.format(cert_path), quiet=True)
-    if result.failed:
+    if result.failed or force:
         run('capsule-certs-generate -v --capsule-fqdn {} '
             '--certs-tar {}'.format(capsule_hostname, cert_path))
```

Comment:
Shouldn't we also check on the result of this command before returning the `cert_path`?
what if it somehow fails?